### DISCUSSION TITLE
Ikea blinds support

### DISCUSSION THREAD

**Discussion by crryp** [sanjoyg/dirigera_platform#100] [Q&A] [answered]
Does this intergration works for all blinds or just the one specified in the documentation: FRYTUR

I would like to know if the: TREDANSEN and the PRAKTLYSING are also supported.

**Comment by sanjoyg** [comment]
I don't have any of the blinds you would have to try. If it doesn't work dump json as per documentation ans will add support

**Comment by crryp** [comment]
Thanks for the fast reaction. I saw the blind: frytur in your documentation as supported. 

I didnt buy them yet since i first want to see if im able to connect with HA. 

So when i do i will get back on if they do work or not!

**Comment by DarthToker420** [comment, marked as answer]
Just confirming that TREDANSEN and PRAKTLYSING cellular blinds work. Wish there was a "stop" function but that seems to be Ikea's shortsightedness
